donna/index.d.ts

```diff
@@ -3,9 +3,8 @@
 // Definitions by: vvakame <https://github.com/vvakame/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare module "donna" {
-    function generateMetadata(modules: string[]): DonnaTypes.Metadata;
-}
+declare function generateMetadata(modules: string[]): DonnaTypes.Metadata;
+export = generateMetadata;
```

Comment:
This should be `export function generateMetadata ...` with no `export = ` declaration